migrations/000020_constraint_normalization.up.sql

```diff
@@ -0,0 +1,352 @@
+BEGIN;
+
+SET search_path = public, pg_catalog;
+
+-- Primary keys and UNIQUE
+
+ALTER TABLE ONLY apps_htcondor_extra
+    DROP CONSTRAINT IF EXISTS apps_htcondor_extra_pkey,
+    DROP CONSTRAINT IF EXISTS apps_htcondor_extra_apps_id_pkey,
+    ADD PRIMARY KEY (apps_id);
+
+ALTER TABLE ONLY authorization_requests
+    DROP CONSTRAINT IF EXISTS authorization_requests_user_id_key;
+
+ALTER TABLE ONLY default_bags
+    DROP CONSTRAINT IF EXISTS default_bags_bag_id_fkey;
+ALTER TABLE ONLY bags
+    DROP CONSTRAINT IF EXISTS bags_id_key; -- duplicated by bags_pkey
+ALTER TABLE ONLY default_bags
+    ADD FOREIGN KEY (bag_id) REFERENCES bags(id) ON DELETE CASCADE;
```

Comment:
This whole deal, and  similar below it, is ugly but apparently sometimes the foreign keys to these tables are relying on the duplicate unique index rather than the primary key index. Which is silly, so I want to change it of course, but I think this is the only way to do that, unlike the `RENAME CONSTRAINT` stuff I can do with actual relied-on-pk-name changes.